server/src/main/java/org/elasticsearch/indices/IndicesService.java

```diff
@@ -239,12 +241,13 @@ public IndicesService(Settings settings, PluginsService pluginsService, NodeEnvi
                           IndexScopedSettings indexScopedSettings, CircuitBreakerService circuitBreakerService, BigArrays bigArrays,
                           ScriptService scriptService, ClusterService clusterService, Client client, MetaStateService metaStateService,
                           Collection<Function<IndexSettings, Optional<EngineFactory>>> engineFactoryProviders,
-                          Map<String, IndexStorePlugin.DirectoryFactory> directoryFactories) {
+                          Map<String, IndexStorePlugin.DirectoryFactory> directoryFactories, ValuesSourceRegistry valuesSourceRegistry) {
```

Comment:
It look like someone was going with one arg per line here.

Reply:
I'm a little confused by this comment - are you saying this should be one line per arg? I think that's what the new auto-formatting would do.

Reply:
When I first looked it at I thought that *was* what it was and you hadn't followed. On second look it wasn't. Sorry! Ignore me.